applications/solvers/dfHighSpeedFoam/rhoYEqn.H

```diff
@@ -50,6 +37,20 @@ const surfaceScalarField phiUc = linearInterpolate(sumYDiffError) & mesh.Sf();
 
             if (!inviscid)
             {
+                hDiffCorrFlux = Zero;
+                diffAlphaD = Zero;
+                sumYDiffError = Zero;
+
```

Comment:
These need to stay at the beginning of the file as you set them once for each timestep. They are accumulated over Yi_s!